src/MoBi.Presentation/Presenter/IPathAndValueBuildingBlockPresenter.cs

```diff
@@ -13,5 +14,6 @@ public interface IPathAndValueBuildingBlockPresenter<in TDTO> : IBreadCrumbsPres
       IEnumerable<ValueFormulaDTO> AllFormulas();
       void AddNewFormula(TDTO expressionParameterDTO);
       IEnumerable<IDimension> DimensionsSortedByName();
+      void SetValueOrigin(TDTO parameterDTO, ValueOrigin valueOrigin);
```

Comment:
I don't get any of the parameter namign here. TDTO? Why is line 15 about expressionParameter and like 17 about parameter. Should it be TParameterDTO maybe? and the like 15 renamed?

Reply:
OMG.